### PR TITLE
OAuth2 param transformations

### PR DESCRIPTION
**Pull request by kunagpal**

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -102,12 +102,18 @@ util = {
...
+                accessTokenUrl: oldParams.accessTokenUrl,
+                clientId: oldParams.clientId,
+                clientSecret: oldParams.clientSecret,
+                clientAuth: oldParams.clientAuth,
+                grantType: oldParams.grantType,
                 scope: oldParams.scope,
-                requestAccessTokenLocally: oldParams.local_access_token
+                username: oldParams.username,
+                password: oldParams.password,
+                redirectUri: oldParams.redirectUri
```
Comment (kamalaknn) [inline lib/util.js]: `tokenType` is missing.

Comment (kamalaknn) [reply lib/util.js]: `accessToken` is missing

Comment (kunagpal) [reply lib/util.js]: 😞

**Comment by kamalaknn** [comment]
LGTM, examples need to updated.

**Pull request merged**